.ipynb_checkpoints/contributing-checkpoint.md

```diff
@@ -0,0 +1,88 @@
+# Guidance on how to contribute
+
```

Comment:
Probably not needed in the PR here (this file is inside the .ipynb_checkpoints folder, which is just a cache folder for jupyter/IPython...)

Reply:
deleted this checkpoint directory. Can we append the gitignore file to deal with this in the future?